src/abstract_hardware_model.h

```diff
@@ -232,7 +232,7 @@ class kernel_info_t {
   //      m_num_cores_running=0;
   //      m_param_mem=NULL;
   //   }
-  kernel_info_t(dim3 gridDim, dim3 blockDim, class function_info *entry);
+  kernel_info_t(dim3 gridDim, dim3 blockDim, class function_info *entry, unsigned long long streamID=0);
```

Comment:
Remove default value and require a streamID always.

Reply:
Addressed in commit e8cc02d